### PR TITLE
Potential exploiters

### PR DESCRIPTION
**Pull request by mauricio1802**

### CODE REVIEW

File: potential-exploiters/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export const createFinding = (deployer: string): Finding => {
+  return Finding.fromObject({
+    name: "Potential Exploiter",
+    description: "A contract was created from a tornado funded account",
+    alertId: "NETHFORTA-28",
+    type: FindingType.Suspicious,
+    severity: FindingSeverity.High,
+    metadata: {
+      deployer: deployer,
+    },
```
Comment (e1Ru1o) [inline potential-exploiters/src/utils.ts:12]: I would like to see the address of the contract too.

File: potential-exploiters/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - Severity is always set to "High".
+  - Type is always set to "Suspicious".
+  - The metadata contains the following fields:
+    - `deployer`: The account funded from Tornado deploying a contract 
+
+## Test Data
+
+The agent behaviour can be verified scanning the following block range
+
+- 13961722 - 13961953. This range include the funded and the deployment related to Defi Dollar Finance incidence `https://twitter.com/defidollar/status/1479718032356241408?s=21`.
```
Comment (e1Ru1o) [inline potential-exploiters/README.md:26]: Maybe you can add more details, for example the address of the contract deployer

**Review by e1Ru1o — APPROVED** [review]

**Review by rachit2501 — APPROVED** [review]

File: potential-exploiters/src/utils.ts (hunk 1 above)
Comment (mauricio1802) [reply potential-exploiters/src/utils.ts:12]: Agree 👍

File: potential-exploiters/README.md (hunk 2 above)
Comment (mauricio1802) [reply potential-exploiters/README.md:26]: I'll add the transaction hashes when the account is funded and when the contract is created 👍

**Pull request merged**